docs/ProtectedBuildGuide.md

```diff
@@ -9,6 +9,7 @@
 	- [Userspace Initialization](#userspace-initialization)
 	- [MPU Initialization](#mpu-initialization)
 	- [Allocation of kernel and user heaps](#allocation-of-kernel-and-user-heaps)
+- [Available APIs](#available-apis)
```

Comment:
can we maintain same tab

Reply:
with which one?

Reply:
This is not a part of `Steps for suporting protected build in a new chipset`.

Reply:
oh It is added in contents, its fine I thought its sub section